docs/guides/platform/ManageInventory.md

```diff
@@ -1,30 +1,24 @@
+The Inventory manages the systems information used for Merlin. It will save the general system information including hostname, type, description and some other information. The detailed inventory info will be changed according to the inventory type. Merlin supports four types of hosts: `IBM i`, `IBM PowerVC`, `IBM Cloud`, and `Jenkins`.  
+
 Go to Merlin GUI and select **Connections -> Inventory** to go to the Inventory page.    
     
 <img src="../../images/guides/Addinventory0.png" width = "800" height = "500" alt="Inventory" align=center />  
 
-All the Inventories which the user created or has permission to will be shown. It includes Name, Hostname, Type, Description, Owner, Provision, Created Time, and Updated Time. The Provision item is a tag that specifies the IBM i server deployed by the Power VC template.  
-- **Add Inventory**   
-Click Add button, enter the Name, Type and Hostname.  After the type is selected, the required info will update.  If one of requirements are not meet,  the +Add button will stay disabled.  
-
-<img src="../../images/guides/Addinventory.png" width = "400" height = "250" alt="Add Inventory" align=center />    
+All the inventories which the user created or has permission to will be shown. It will display the `Name`, `Hostname`, `Type`, `Description`, `Owner`, `Provision`, `Created Time`, and `Updated Time`. The `Provision` item is a tag that specifies the IBM i server deployed by the Power VC template.  
 
-Specify all necessary fields and click +Add button, then the new inventory record will be imported.   
-For the security connections, Merlin supports Test TLS function, that allow customer to accept the remote CA. Such as Power VC Inventory.
+- **Add Inventory** - Click the `+ Add` button and enter the `Name`, `Type` and `Hostname`.  After the type is selected, the required fields will updated. If one of requirements are not meet,  the `+ Add` button will stay disabled. Specify all the necessary fields and click the `+ Add` button so then the new inventory record will be imported to the Inventory table.
 
-<img src="../../images/guides/Pvcinventory.png" width = "400" height = "300" alt="Add power vc inventory" align=center />  
+  <img src="../../images/guides/Addinventory.png" width = "400" height = "250" alt="Add Inventory" align=center />    
 
-Specify the Hostname and Port, the Test TLS button will be enabled, and it will catch the Power VC CA.  In the detailed certificate windows, click the Accept button.  Then this CA be added to Merlin trust store.
+  For the security connections, Merlin supports the `Test TLS` function which allows customers to accept the remote CA such as for a Power VC Inventory. After specifying the `Hostname` and `Port`, the `Test TLS` button will be enabled and it will catch the Power VC CA. In the detailed certificate windows, click the `Accept` button. Then this CA be added to the Merlin trust store.
```

Comment:
'For the security connections', does this mean 'secure connection'?

Reply:
Yes, that was a typo. Fixed it